tests/Mocks/IDEntityCantAutodetectMock.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AvtoDev\IDEntity\Tests\Mocks;
+
+use AvtoDev\IDEntity\Types\AbstractTypedIDEntity;
+
+class IDEntityCantAutodetectMock extends AbstractTypedIDEntity
```

Comment:
Why not Mockerty?

Reply:
Because we can't set protected property in Mockery

Reply:
Nope. [We can](https://sandro-keil.de/blog/2015/08/31/phpunit-with-peek-and-poke-no-more-reflection/)